### PR TITLE
Testexample

### PR DESCRIPTION
**Pull request by alkasubin**
Added Text in Readmefile

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,5 +11,4 @@ Claire Oct-8-2020 line 2
 Claire Oct-15-2020 line 3
 Ginn's change: Hopefully you can master Git soon.
 
-
-This is from demo on Mar 17th
\ No newline at end of file
+Created new brach for next demo-Alka
```
Comment (GinnWang) [inline README.md]: please add more explanation for the demo

**Review by GinnWang — CHANGES_REQUESTED** [review]
this is conversation #2

File: README.md (hunk 1 above)
Comment (alkasubin) [reply README.md]: updated readmefile as per review

**Review by GinnWang — APPROVED** [review]
Code looks good.

**Pull request merged**